instant-xml-macros/src/de.rs

```diff
@@ -11,22 +11,53 @@ pub(crate) fn from_xml(input: &syn::DeriveInput) -> TokenStream {
         Err(e) => return e.to_compile_error(),
     };
 
+    let name = match &meta.rename {
```

Comment:
I would like to split this refactoring off into a separate commit.

Reply:
I can move `default_namespace` and `name` back to the `deserialize_struct` function, as ultimately they're not needed here, so the minimum changeset becomes cleaner.

As a separate PR it would probably make sense to split these into a struct and define `(de)?serialize_{enum,struct}` functions on that parameter struct. WDYT?